vertx-db2-client/src/test/java/io/vertx/db2client/tck/DB2TransactionTest.java

```diff
@@ -62,5 +65,11 @@ protected void cleanTestTable(TestContext ctx) {
   protected String statement(String... parts) {
     return String.join("?", parts);
   }
+  
+  @Test
+  public void testDelayedCommit(TestContext ctx) {
+    assumeFalse("DB2 on Z holds write locks on inserted columns with isolation level = 2", rule.isZOS());
```

Comment:
Why fail the test for Z instead of wrapping it with 
if (!rule.isZOS()) {
  super.testDelayedCommit(ctx);
}

Reply:
Here I am using `assumeFalse` which is slightly different than `assertFalse`. The `assumeFalse` method will simply bail out of the test if the assumption is not met, which shows up as a SKIP in the JUnit reports. So it's effectively the same thing as wrapping the test in the `if` statement you suggested, except the test will show up as SKIPPED instead of PASSED in the report.